easybuild/easyconfigs/d/Dakota/Dakota-6.16.0-boost177-dllimport.patch

```diff
@@ -0,0 +1,15 @@
+# Author: Jiri Furst <jiri.furst@gmail.com>
+# Boost dll::import was renamed to dll::import_symbol  
+# see https://github.com/boostorg/dll/commit/6fed722419caec8ff1049ea0e78afcd75daa6494
```

Comment:
This commit only changes some `std::getline` stuff, I guess you copied the wrong commit URL and it should be https://github.com/boostorg/dll/commit/6b2e3c7f660199f2bb0d69b667a38ee21cd49626 instead? 
(If so, note that you will also need to recreate the checksum of the patch file)